components/runtime/vala/Makefile

```diff
@@ -45,6 +52,6 @@ install:	$(INSTALL_32_and_64)
 
 test:		$(TEST_32_and_64)
 
-BUILD_PKG_DEPENDENCIES =	$(BUILD_TOOLS)
-
-include ../../make-rules/depend.mk
+REQUIRED_PACKAGES += SUNWcs
```

Comment:
I had checked this (following your receipt): dependencies detected via grep in configure.ac are glib-2.0, gobject-2.0, gmodule-2.0 and gthread-2.0.
I concluded that I don't have to add other dependencies than the generated.